src/shipping/strategies/googlepay-braintree-shipping-strategy.ts

```diff
@@ -0,0 +1,26 @@
+import { ConsignmentActionCreator, ShippingRequestOptions } from '../';
+import { AddressRequestBody } from '../../address';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../checkout';
+
+import { ShippingStrategy } from './';
+
+export default class GooglePayBraintreeShippingStrategy extends ShippingStrategy {
```

Comment:
A question, why do we need this? It seems it looks exactly same as `DefaultShippingStrategy`? We usually don't create a new one unless it deviates from the default strategy.